src/contract.rs

```diff
@@ -73,9 +76,120 @@ pub fn execute(
             validator_oper_addr,
         } => remove_validator(deps, info, validator_oper_addr),
         ExecuteMsg::WithdrawFunds { amount } => withdraw_funds(deps, info, amount),
+
+        ExecuteMsg::DeleteMetricsForTimestamp {
+            timestamp,
+            validator_idx,
+            validator_ct,
+        } => delete_metrics_for_timestamp(
+            deps,
+            info,
+            timestamp,
+            validator_idx as usize,
+            validator_ct as usize,
+        ),
+
+        ExecuteMsg::DeleteMetricsForValidator {
+            validator_opr_addr,
+            timestamp_idx,
+            timestamp_ct,
+        } => delete_metrics_for_validator(
+            deps,
+            info,
+            validator_opr_addr,
+            timestamp_idx as usize,
+            timestamp_ct as usize,
+        ),
+
+        ExecuteMsg::RemoveTimestamp { timestamp_idx } => delete_timestamp(deps, info, timestamp_idx)
```

Comment:
would prefer to have the same name for the fn. `remove_timestamp`

Reply:
Got it. Good catch!